openmetadata-docs/content/v1.3.x/connectors/ingestion/lineage/index.md

```diff
@@ -65,6 +65,10 @@ to identify the graph nodes as OpenMetadata Entities.
 
 Note that if a Model is not materialized, its data won't be ingested.
 
+### Cross Project Lineage
+
+We support cross-project lineage, but the data must be ingested within a single service. This means you need to perform lineage ingestion for just one service while including multiple projects.
+
```

Comment:
@Prajwal214 this is only specific to bigquery, we should not include this in this general lineage doc

Reply:
I have updated in the BigQuery Docs